collectors/python.d.plugin/example/example.conf

```diff
@@ -61,8 +61,27 @@
 #
 # Additionally to the above, example also supports the following:
 #
-# - none
+#     num_lines: 4            # the number of lines to create
+#     lower: 0                # the lower bound of numbers to randomly sample from
+#     upper: 100              # the upper bound of numbers to randomly sample from
 #
 # ----------------------------------------------------------------------
 # AUTO-DETECTION JOBS
-# only one of them will run (they have the same name)
+
+four_lines:
+    name: "Four Lines"      # the JOB's name as it will appear at the
+    update_every: 1         # the JOB's data collection frequency
+    priority: 60000         # the JOB's order on the dashboard
+    penalty: yes            # the JOB's penalty
+    autodetection_retry: 0  # the JOB's re-check interval in seconds
+    num_lines: 4            # the number of lines to create
+    lower: 0                # the lower bound of numbers to randomly sample from
+    upper: 100              # the upper bound of numbers to randomly sample from
```

Comment:
Why do you think we need to copy comments?


https://github.com/andrewm4894/netdata/blob/8482d276e2eb6f2b8736d5c560c5c126abc4f801/collectors/python.d.plugin/example/example.conf#L53-L79

Reply:
to be as clear as possible for the user. e.g. most will probably just start by scanning the actual lines that are not comments in the file and then see what they are as opposed to reading the green wall of text that is all the stuff above. 

i scan the conf with my eyes - see what lines are not comments, see what they do and then got 90% of what i need to be productive. Then as needed at a later stage i may go back and read every single line of the conf comments and all.